### PR TITLE
[PPML] Add LR example to occlum Spark

### PR DESCRIPTION
**Pull request by piaolaidelangman**

### CODE REVIEW

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor_spark-logisticregressionexample.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor_spark-logisticregressionexample.yaml:1]: We can re-used existing executor.yaml

Comment (piaolaidelangman) [reply ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/executor_spark-logisticregressionexample.yaml]: Yes. I use command `sed` to replace `container name` in `executor.yaml` in run_.sh in this pr.

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md [hunk 2]
```diff
@@ -20,10 +20,23 @@ bash build-docker-image.sh
 
 ## Run Spark executor in Occlum:
 
+### Run Spark.pi example
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md:23]: Change to SparkPI

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md [hunk 3]
```diff
@@ -20,10 +20,23 @@ bash build-docker-image.sh
...
+
 1. Download [Spark 3.1.2](https://archive.apache.org/dist/spark/spark-3.1.2/spark-3.1.2-bin-hadoop2.7.tgz), and setup `SPARK_HOME`. Or set `SPARK_HOME` in `run_spark_pi.sh`.
 2. Modify `${kubernetes_master_url}` to your k8s master url in the `run_spark_pi.sh `
 3. Modify `executor.yaml` for your need
 
 ```bash
 ./run_spark_pi.sh
 ```
+
+### Run Spark.LogisticRegression example
```
Comment (qiyuangong) [inline ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md:33]: Spark ML LogisticRegression example

**Comment by piaolaidelangman** [comment]
I have changed the example name.

File: ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md (hunk 3 above)
Comment (piaolaidelangman) [reply ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/README.md]: Ok.

**Review by qiyuangong — APPROVED** [review]
LGTM

**Pull request merged**